features/mesh/services/Item.feature

```diff
@@ -96,40 +90,26 @@ Feature: mesh / services / item
       Then the "$input-search" element isn't disabled
       And I wait for 500 ms
       When I "type" "name:a-service protocol:tcp" into the "$input-search" element
-      And the URL "/meshes/default/dataplanes+insights" responds with
-        """
-        """
```

Comment:
These weird empty "set up the Cypress spy" steps are no longer needed as we maintain a full history of requests made during the test and use that for looking for the request we are interested in.